tools/autograd/derivatives.yaml

```diff
@@ -1129,6 +1129,9 @@
 - name: view(Tensor(a) self, int[] size) -> Tensor(a)
   self: grad.reshape(self.sizes())
 
+- name: view.dtype(Tensor(a) self, ScalarType dtype) -> Tensor(a)
+  output_differentiability: [False]
```

Comment:
That will work for now. You can ping me if you want to change that to True in the future :)

Reply:
Is the grad of this overload well defined mathematically? For the reinterpreting of two different types?

The int<-->float reinterpret will not work because int tensors do not support gradients. The only reinterpret in question I think is the `double<-->complex64`. But I don't think this makes sense either. For example, if we change the real part of the complex64 from 0 to 1.1111111111*2^-80, then the exponent bits of the double tensor will be changed, and the limit
```
 lim   (f(x+dx) - f(x)) / dx
dx->0
```
don't seem to be converging.

Reply:
Well, the forward is really defined mathematically either right? haha
But I do agree that we most likely want to keep it non differentiable for now.